lib/omise/res/OmiseApiResource.php

```diff
@@ -250,7 +250,7 @@ private function _executeTest($url, $requestMethod, $key, $params = null)
      */
     private function genOptions($requestMethod, $userpwd, $params)
     {
-        $user_agent        = "OmisePHP/".OMISE_PHP_LIB_VERSION;
+        $user_agent        = "OmisePHP/" . OMISE_PHP_LIB_VERSION . " PHP/" . phpversion();
```

Comment:
It seems that current coding convention does not have spaces between dot (.) used for concatenation. Does this matter?